app/views/external_submissions/index.html.erb

```diff
@@ -1 +1,18 @@
-<h1>External submissions</h1>
+<% breadcrumb :external_submissions %>
+
+<%= mltop_title "External submissions" %>
+
+<% @models.each do |model| %>
+  <div class="overflow-hidden bg-gray-50 sm:rounded-lg sm:shadow mb-8">
+    <div class="border-b border-gray-200 px-4 py-5 sm:px-6">
+      <div class="-ml-4 -mt-2 flex flex-wrap items-center justify-between sm:flex-nowrap">
+        <div class="ml-4 mt-2">
+          <h3 class="text-base font-semibold leading-6 text-gray-900"><%= model %></h3>
+        </div>
+      </div>
+    </div>
+    <div role="list">
+      <%= render partial: "hypothesis", collection: model.hypothesis, as: :hypothesis %>
+    </div>
```

Comment:
This is because of the path where partial is searched. When

```erb
    <div role="list">
      <%= render model.hypothesis %>
    </div>
```

is used it is trying to find the partial in the following location: `hypotheses/_hypothesis` instead of looking at the beginning in the current view context. when the `partial:` keyword is used, it starts to search partial in the current directory, and this is what we want.